frame/support/procedural/src/lib.rs

```diff
@@ -50,6 +50,9 @@ use proc_macro::TokenStream;
 /// prefix. Instance prefix is "" for default instance and "Instance$n" for instance number $n.
 /// Thus, instance 3 of module Example has a module prefix of `Instance3Example`
 ///
+/// **Warning**: storage doesn't support inserting empty values, so the value must encode to
```

Comment:
We should probably make this a sub-heading as well to make it more prominent, otherwise it may goes down in the overall text?

Reply:
I added a section Warning at the end and kept this one.